Algorithms/Medium/62_UniquePaths/Solution.py

```diff
@@ -0,0 +1,12 @@
+class Solution:
+    def uniquePaths(self, m: int, n: int) -> int:
+        
+        bottomRow = [1]*n
+        
+        for column in range(m - 1):
+            newRow = [1]*n
```

Comment:
use snake case and also add spaces between operator and operand
also, add a blank line below this line